### PR TITLE
Api routes 1/3- refactored my /booking endpt to initialize Booking Service with data from Overview API

### PR DESCRIPTION
**Pull request by hannahmanfredi**

### CODE REVIEW

File: client/src/Booking.jsx [hunk 1]
```diff
@@ -101,10 +106,31 @@ styleLink.rel = "stylesheet";
 styleLink.href = "https://cdn.jsdelivr.net/npm/semantic-ui/dist/semantic.min.css";
```
Comment (djian001) [inline client/src/Booking.jsx:134]: this is interesting! why is this added here instead of the HTML?

File: client/src/Booking.jsx [hunk 2]
```diff
@@ -101,10 +106,31 @@ styleLink.rel = "stylesheet";
...
 document.head.appendChild(styleLink);
 
+class BookingButton extends React.Component {
```
Comment (djian001) [inline client/src/Booking.jsx:137]: Might want to move this to a new file for cleaner code?

File: package.json [hunk 3]
```diff
@@ -12,19 +12,20 @@
   },
   "scripts": {
     "start": "nodemon ./server/index.js",
-    "react-dev": "webpack --config webpack.config.js",
+    "altStart": "NODE_ENV=production node ./server/index.js",
```
Comment (djian001) [inline package.json:15]: this is interesting, need to look into what this does

File: server/server.js [hunk 4]
```diff
@@ -1,23 +1,111 @@
...
 const app = express();
 
+//allow my service to accept every other proxy
+const origins = ['http://localhost:2000',
+'http://localhost:5500',
+'http://localhost:3000'];
+  // add michael's proxy port numb when avail.
+
+app.use(cors());
+app.use(cors({ origin: origins }));
```
Comment (djian001) [inline server/server.js]: from my understanding, if you use app.use(cors()), it allows access to everything so you don't need to double use the app.use(cors({ origin: origins }));

**Review by djian001 — APPROVED** [review]

File: client/src/Booking.jsx (hunk 1 above)
Comment (hannahmanfredi) [reply client/src/Booking.jsx:134]: just another way of doing it!

File: client/src/Booking.jsx (hunk 2 above)
Comment (hannahmanfredi) [reply client/src/Booking.jsx:137]: yea it's here for v1 but def will move to its own file, good pt

File: package.json (hunk 3 above)
Comment (hannahmanfredi) [reply package.json:15]: that's so i can serve either my proxy or my service!

File: server/server.js (hunk 4 above)
Comment (hannahmanfredi) [reply server/server.js]: Yup, this is fixed in a diff PR. TY
